installation/routines/setup_jukebox_webapp.sh

```diff
@@ -56,9 +55,11 @@ _jukebox_webapp_build() {
 
 _jukebox_webapp_download() {
   echo "  Downloading web application" | tee /dev/fd/3
+  local JUKEBOX_VERSION=$(get_version_string "${INSTALLATION_PATH}/src/jukebox/jukebox/version.py")
   local TAR_FILENAME="webapp-build.tar.gz"
+
   cd "${INSTALLATION_PATH}/src/webapp" || exit_on_error
-  _download_file_from_google_drive ${GD_ID_COMPILED_WEBAPP} ${TAR_FILENAME}
+  wget --quiet ${GIT_URL}/releases/download/v${JUKEBOX_VERSION}/webapp-v${JUKEBOX_VERSION}.tar.gz -O ${TAR_FILENAME}
```

Comment:
GIT_URL will not work for Forks, as the tags / releases are not available.
Maybe use "https://github.com/${GIT_UPSTREAM_USER}/${GIT_REPO_NAME}" ?

But GIT_UPSTREAM_USER must not be changed then, but is currently possible.

Reply:
> GIT_URL will not work for Forks, as the tags / releases are not available.

Good catch. I will fix this!

> Also there is currently a inconsistency in the tag naming.

Standard in most Git repos is lowercase: e.g. `v.3.4.0`. I prefer to stick to that.